### DISCUSSION TITLE
关于下载的漫画存储位置问题

### DISCUSSION THREAD

**Discussion by Eustia-gif** [xiaoyaocz/flutter_dmzj#196] [Q&A] [answered]
找了好久没发现下载的漫画的存储位置，是文件存储方式或者存储位置无法访问的原因吗

**Comment by EndRedIric** [comment, marked as answer]
作者做第三方客户端为了保证下载的资源不外流，储存位置还是藏的很深的
没root建议不要找了，找不到的

**Comment by FengTeLai** [comment]
所以到底在哪个文件夹下呢，ROOT了

**Comment by xiaoyaocz** [comment]
/data/user/0/com.xycz.dmzjx/files
